apis/c/operator/build.rs

```diff
@@ -2,10 +2,11 @@ use std::path::Path;
 
 fn main() {
     dora_operator_api_types::generate_headers(Path::new("operator_types.h"))
-        .expect("failed to create operator_api.h");
+        .expect("failed to create operator_types.h");
 
     // don't rebuild on changes (otherwise we rebuild on every run as we're
     // writing the `operator_types.h` file; cargo will still rerun this script
     // when the `dora_operator_api_types` crate changes)
-    println!("cargo:rerun-if-changed=");
+    println!("cargo:rerun-if-changed=../../rust/operator/types/src/lib.rs");
```

Comment:
The thing is that `src/lib.rs` seems to always be changing due to the procedural macro. And so linking the exact source file make it compile the header file only if it actually was changed. Reducing compilation time from 8 sec -> <1sec, if nothing changed.

Reply:
I tried to fix this with an empty `rerun-if-changed` instruction, but apparently the [official recommendation](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed) is to do `cargo:rerun-if-changed=build.rs` in such cases.

I don't think that the relative path into another crate is a good idea because it only works when the repo is checked out, not when using `dora-operator-api-c` from crates.io.